src/cmd-init

```diff
@@ -4,6 +4,21 @@ set -xeuo pipefail
 dn=$(dirname $0)
 . ${dn}/cmdlib.sh
 
+source="${1:-}"
+if [ -z "${source}" ] && ! [ -e src/config ]; then
+    set +x
+    cat 1>&2 <<EOF
+Usage: coreos-assembler init GITCONFIG
+
+  For example, you can use https://github.com/cgwalters/fedora-coreos-config
+  as GITCONFIG, or fork it.  Another option useful for local development
+  is to pass a file path starting with '/' - a symlink to it will be created
```

Comment:
will a symlink work? I don't think it will if the place where the symlink points isn't also accessible inside the container.

Reply:
Yeah...it will inside a pet container (or using `shell` in this container).  Dunno how much we want to document that yet.  I guess I could just drop this text for now?

Reply:
Updated the help to clarify that a bit.  I'm not super worried about this right now though as the `README.md` doesn't mention it and so only people who are looking for it will find it.